testsuite/MDAnalysisTests/test_api.py

```diff
@@ -49,3 +50,33 @@ def test_ResidueGroup():
 
 def test_SegmentGroup():
     assert mda.SegmentGroup is mda.core.groups.SegmentGroup
+
+
+@pytest.mark.parametrize('module',
```

Comment:
My issue with such a list is that new modules are likely to not be added. What about walking the tree and find the modules? It might be done with importlib, but the simplest would be to check the directory tree for `__init__.py` files.

Reply:
Walking the complete package works fine and is a good idea,.